dali/imgcodec/decoders/nvjpeg2k/nvjpeg2k.cc

```diff
@@ -248,8 +248,10 @@ DecodeResult NvJpeg2000DecoderInstance::DecodeImplTask(int thread_idx,
   result.success = DecodeJpeg2000(in, decode_out.raw_mutable_data(), ctx);
 
   if (is_processing_needed) {
+    DALIImageType out_format = opts.format;
```

Comment:
This will be fixed in a different way. Maybe you can revert this one for now

Reply:
Reverted